analytics/config/config.go

```diff
@@ -0,0 +1,48 @@
+package config
+
+import (
+	"github.com/golang/glog"
+	"github.com/prebid/prebid-server/analytics"
+	"github.com/prebid/prebid-server/analytics/filesystem"
+	"github.com/prebid/prebid-server/config"
+)
+
+//Modules that need to be logged to need to be initialized here
+func NewPBSAnalytics(analytics *config.Analytics) analytics.PBSAnalyticsModule {
+	modules := make(enabledAnalytics, 0)
+	if len(analytics.File.Filename) > 0 {
```

Comment:
Is it possible for analytics.File to be null? Looking at the definition, I don't think so, as there are no pointers in that chain. Might be worth a test if you haven't tested yet.

Reply:
It can't. Go won't compile if you say `struct.NonPointerField = nil`, because `nil` isn't assignable to structs. If you don't set it, you get the zero object-- a non-nil struct whose properties are all zero objects.